app/uk/gov/hmrc/agentmapping/connector/EnrolmentStoreProxyConnector.scala

```diff
@@ -77,24 +74,21 @@ class EnrolmentStoreProxyConnector @Inject()(
 
   //ES2 - delegated
   private def getDelegatedEnrolmentsCountFor(userId: String, startRecord: Int, service: String)(
-    implicit hc: HeaderCarrier): Future[(Int, Int)] =
-    monitor(s"ConsumedAPI-ESPes2-$service-GET") {
-
-      def url(userId: String, startRecord: Int, service: String): URL =
-        new URL(
-          baseUrl,
-          s"enrolment-store-proxy/enrolment-store/users/$userId/enrolments?type=delegated&service=$service&start-record=$startRecord&max-records=$batchSize"
-        )
-
-      http.GET[EnrolmentResponse](url(userId, startRecord, service).toString)(responseHandler, hc, ec).map {
-        enrolmentResponse =>
-          val filteredCount =
-            enrolmentResponse.enrolments
-              .count(e => e.state.toLowerCase == "activated" || e.state.toLowerCase == "unknown")
-
-          (enrolmentResponse.enrolments.length, filteredCount)
-      }
+    implicit hc: HeaderCarrier): Future[(Int, Int)] = {
+
+    def url(userId: String, startRecord: Int, service: String): String =
```

Comment:
We used to return a URL, but now we return a String - why?  This method is called url(...)